### PR TITLE
ISS-75 Split procedures by morning and night

### PR DESCRIPTION
**Pull request by EvgSem**

### CODE REVIEW

File: FaceCareRoutine/ProcedureFactory.swift [hunk 1]
```diff
@@ -44,26 +44,44 @@ class ProcedureFactory {
...
-                                howOften : self.howOftenMustBeDone(procedureType: ProcedureType(rawValue: procedureDescription.type!)!),
-                                procedureDescription: procedureDescription, partOfTheDay: partsOfDay[DayPart.night]!, steps:  NSSet(array: steps), product: product)
-                            
-                            self.procedures.append(procedure)
-                        }
+                    let howOften = self.howOftenMustBeDone(procedureType: ProcedureType(rawValue: procedureDescription.type!)!)
+                    let isMorning = self.isItMorningProcerude(procedureType: ProcedureType(rawValue: procedureDescription.type!)!)
+                    let isNight = self.isItNightProcerude(procedureType: ProcedureType(rawValue: procedureDescription.type!)!)
+                    
+                    if isMorning && isNight {
```
Comment (sbondini) [inline FaceCareRoutine/ProcedureFactory.swift]: this long if could be reduced

File: FaceCareRoutine/ProcedureFactory.swift [hunk 2]
```diff
@@ -121,6 +154,60 @@ class ProcedureFactory {
         }
     }
     
+    private func isItMorningProcerude(procedureType: ProcedureType) -> Bool {
```
Comment (sbondini) [inline FaceCareRoutine/ProcedureFactory.swift]: typo

File: FaceCareRoutine/ProcedureFactory.swift [hunk 3]
```diff
@@ -121,6 +154,60 @@ class ProcedureFactory {
...
+            case .eyeCreamType:
+                return true
+            case .moiterazerType:
+                return true
+            case .sunProtectionType:
+                return true
+        }
+    }
+    
+    private func isItNightProcerude(procedureType: ProcedureType) -> Bool {
```
Comment (sbondini) [inline FaceCareRoutine/ProcedureFactory.swift]: typo

File: FaceCareRoutine/ProcedureFactory.swift (hunk 1 above)
Comment (EvgSem) [reply FaceCareRoutine/ProcedureFactory.swift]: reduced

**Pull request merged**